src/utils/useNumber.ts

```diff
@@ -6,8 +6,11 @@ const getLocaleToUse = (locale?: string) => (locale === 'gx' ? 'fr' : locale ||
  * formatter
  */
 export const useNumberFormatter = (): any => {
-  const formatter = (locale?: string): any => {
-    const localeToUse = getLocaleToUse(locale);
+  const formatter = (locale?: string, countryCode?: string): any => {
+    let localeToUse = getLocaleToUse(locale);
```

Comment:
i think the country code is in the user's locale
the `getlocaletouse` strips off the country code

Reply:
ugh, nvm, i am wrong here.. apologies.